.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -34,27 +34,26 @@ _What changed calling `firejail --noprofile /path/to/program` in a terminal?_
 
 ### Additional context
 
-_Any other context you have to understand the problem_
+_Any other detail that may help understand/debug the problem_
```

Comment:
I'm not a native english speak, but I would place a "to" here. Just to make sure it is not missing by accident.
```suggestion
_Any other detail that may help to understand/debug the problem_
```

Reply:
> I'm not a native english speak, but I would place a "to" here. Just to make
> sure it is not missing by accident.

I'm not either, but I remember searching about this a while back, and from what
I remember both forms are accepted.

Using "to" here sounds a bit formal IMHO, but it works too.  It feels like the
difference would be which parts of the sentence "combine", like "that may help
understand [pause] the problem" vs "that may help [pause] to understand the
problem".